server.js

```diff
@@ -3636,6 +3674,11 @@ function fetchFromSvg(request, url, cb) {
   });
 }
 
+function ordinalNumber(n) {
+  var s=["ᵗʰ","ˢᵗ","ⁿᵈ","ʳᵈ"], v=n%100;
```

Comment:
That's pretty cool, but I fear it won't render well:
![nth](https://cloud.githubusercontent.com/assets/100689/7445323/ee809eda-f1ad-11e4-9609-a430a408892a.png)